.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.linting.pylintEnabled": true
+}
```

Comment:
Please, remove this file.

See https://docs.github.com/es/github/using-git/ignoring-files, you might want to set up a global rule in your system to exclude those files, unless you are working on a project that keeps track of those.